### ISSUE TITLE
Credentials instructions do not match Google help or configuration

### ISSUE THREAD

**Issue by TotallyInformation** [node-red/node-red-web-nodes#228]
### Which node are you reporting an issue on?

node-red-node-google

### What are the steps to reproduce?

Add calendar node, attempt to configure credentials.

### What happens?

None of the instructions match wording in either the referenced Google help page nor either of the 2 Google configuration sites (developer console & API settings).

### What do you expect to happen?

To be able to follow the instructions for setting up the Google API in the node via the Google help pages.

### Please tell us about your environment:

- [x] Node-RED version: 0.19.1
- [x] node.js version: v8.11.3
- [x] npm version: 5.6.0
- [x] Platform/OS: Windows 10 1709
- [x] Browser: Chrome

It has been quite a long time since I tried to set up a Google API connection in Node-RED and I can't follow the instructions at all since they don't at all match the Google sites referenced. I'm sure that beginners will have even more issues.

**Comment by Marooned-MB** [comment]
Yup. I managed to pass some steps but now I'm stucked at allowing my app to access Google Account data and then Google returns to http://127.0.0.1:1880/google-credentials/auth/callback?state=... with error: `auth worked but profile fetching failed`.

**Comment by dceejay** [comment]
a PR would be gratefully received.

**Comment by Marty56** [comment]
I am also totally lost by these poor instruction.
Im my case it ended with an error message that 'some profile could not be reached" (on exact wording).
I also fiddled around by entering the callback uri  (https://node-red.example.com) somewhere in these convoluted google api interfaces. At the end I received an email from Google that they could not verify those uri because they were not online. No surprise I am running the node red server locally, no ports open and could have even switched it off when they were testing.

It is too sad that the level of documentation does not allow a 'none Google authentication expert' to use this node. 
Maybe it is not working at all because if Google wants to verify the callback uri this would mean that my node red server would be exposed to the internet. Which would be a true show stopper for me.

**Comment by kneibdev** [comment]
No solution for this? Drives me crazy....

**Comment by dceejay** [comment]
If anyone makes this work and has the time to update the instructions we would gladly accept a Pull Request. Thanks

**Comment by michaelblight** [comment]
I already have this working from some time ago, but ended up here when I got the "auth worked but profile fetching failed" error trying to set up a second instance. I have two Node Reds running in Docker containers on one IP; one with a Calendar node already configured, and the other I'm trying to configure. Given the Authorised Redirect URL differs for each (just the port), I thought it would work setting up a second. But perhaps that was too optimistic. 

I'd be happy to help with the documentation - but to which are you referring? The link in the config is to a Google page. Are you talking about fleshing out the node's page under pre-requisites?

**Comment by kneibdev** [comment]
Have exact problem like @michaelblight ,

Works on existing machine but can't auth on new one. For the moment I wrote a simple nodejs script and call this with 'exec node'. 

But I am surprised that this issue isn't fixed because for me it is one of the most important functions: adding things in my calendar.
I wish I have the knowledge to fix this...

**Comment by Gerben321** [comment]
Same problem as @michaelblight and @kneibdev . Enabled two Google+ api's and the legacy People API and the Calendar API and I still get "auth worked but profile fetching failed" at the end of all the login screens. I can't imagine we're the only ones and that there's no solution? The instructions are also VERY limited so there's a lot of things that can go wrong here.
